src/main/java/org/fundacionjala/at15/pacman/Movement.java

```diff
@@ -0,0 +1,136 @@
+package org.fundacionjala.at15.pacman;
+
+import static org.fundacionjala.at15.pacman.Constants.Board.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.Timer;
+import javax.swing.ImageIcon;
+
+public class Movement {
+    private int[][] board1; // 0 = empty, 1 = wall, 2 = pacman, 3 = ghost, 4 = dot
+    private int[][] auxBoard;
+    private JLabel[][] matriz;
+    private Timer timerG;
+
+    private int up = 0;
+    private int down = 0;
+    private int left = 0;
+    private int right = 0;
+
+    public void insertBoard(JPanel gamePanel) {
+        String aux = ".png";
+        int auxUp = getUp();
+        int auxDown = getDown();
+        int auxLeft = getLeft();
+        int auxRight = getRight();
+        for (int indI = 0; indI < board1.length; indI++) {
+            for (int indJ = 0; indJ < board1.length; indJ++) {
+                if (board1[indI][indJ] == 2) {
```

Comment:
Should this block replaced by using UtilitiesPacman as well?

Reply:
The Movement.java class was a refactor that I was doing at the beginning and then I converted it to the UtilitiesPacman.java class. I forgot to delete that file, sorry.

Reply:
No worries, thanks for clarification